docs/Toy/toy/dialects/toy.py

```diff
@@ -3,58 +3,47 @@
 """
 
 from __future__ import annotations
-from abc import ABC, abstractmethod
 
+from abc import ABC, abstractmethod
 from typing import TypeAlias, cast
 
-from xdsl.ir import (
-    Dialect,
-    Operation,
-    SSAValue,
-    Attribute,
-    Block,
-    Region,
-    OpResult,
-)
-
 from xdsl.dialects.builtin import (
+    DenseIntOrFPElementsAttr,
     Float64Type,
     FunctionType,
+    StringAttr,
```

Comment:
Can you please adjust your IDE/linters/etc to avoid reordering imports?
It adds a lot of noise that is unrelated to the changes in this PR

Reply:
Addressed in #1243. Rebased to main.